apps/nextjs-website/react-components/types/HeroCounter/HeroCounter.types.ts

```diff
@@ -0,0 +1,14 @@
+import { type CommonProps, type Generic } from '../common/Common.types';
+
+export interface HeroCounterProps extends CommonProps, HeroCounterTextProps {
+  readonly background?: string | Generic;
+  readonly counterNumber?: number;
+  readonly counterText?: string;
+}
+
+export interface HeroCounterTextProps extends CommonProps {
+  readonly title: string;
+  readonly subtitle?: string | Generic;
```

Comment:
```suggestion
  readonly subtitle?: string | JSX.Element;
```
Consistency with other similar field across different components (like Editorial).